TV by the Numbers.js

```diff
@@ -1,144 +1,208 @@
-	if (!res || !res.stringValue)
-		return null;
-	return Zotero.Utilities.trim(res.stringValue);
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function(items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function getSearchResults(doc, checkOnly) {
+	var items = {};
+	var found = false;
+	var rows = ZU.xpath(doc, '//div[@class="container container-small"]/article/h2/a');
+	for (var i = 0; i < rows.length; i++) {
+		var href = rows[i].href;
+		var title = ZU.trimInternal(rows[i].textContent);
+		if (!href || !title) continue;
+		if (checkOnly) return true;
+		found = true;
+		items[href] = title;
+	}
+	return found ? items : false;
 }
 
-function xpre(doc, node, xpath, reg) {
-	var xpmatch = xpath_string(doc, node, xpath);
-	return reg.exec(xpmatch)[1];
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		Zotero.selectItems(getSearchResults(doc, false), function(items) {
+			if (!items) {
+				return true;
+			}
+			var articles = [];
+			for (var i in items) {
+				articles.push(i);
+			}
+			ZU.processDocuments(articles, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
 }
 
 function scrape(doc, url) {
-	var items = new Array();
-	var posts = doc.evaluate("//div[@class='post-alt blog']", doc, null,
-		XPathResult.ANY_TYPE, null);
-		
-	var post_count = 0;
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+	translator.setDocument(doc);
 
-	while (post = posts.iterateNext()) {
-		var newItem = new Zotero.Item("webpage");
-		newItem.publicationTitle = PUB_TITLE;
-		
-		var link = post.getElementsByTagName("a")[0];
-		newItem.url = link.href;
-		
-		var title = Zotero.Utilities.unescapeHTML(
-			Zotero.Utilities.cleanTags(link.textContent));
-		title = title.replace(/(\s+)(?:‘|’)|(?:‘|’)(\s+)/g, "$1''$2").replace(/‘|’/g, "'");
-		newItem.title = title;
-		
-		if (XPATH_DATE)
-			newItem.date = xpre(doc, post, XPATH_DATE, RE_DATE);
-		if (XPATH_PAGES)
-			newItem.pages = xpath_string(doc, post, XPATH_PAGES);
-		
-		//authors
-		var author_text = xpre(doc, post, XPATH_AUTHORS, RE_AUTHORS);
-		var authors = [];
-		if (author_text) {
-			if (author_text.indexOf(" and ") != -1)
-				authors = author_text.split(" and ");
-			else if (author_text.indexOf(";") != -1)
-				authors = author_text.split(";");
+	translator.setHandler('itemDone', function(obj, item) {
+		item.date = ZU.strToISO(item.date);
+		var authors = ZU.xpath(doc, '//a[@rel="author"]');
+		for (var i in authors) {
+			if (authors[i].textContent == 'TV By The Numbers')
+				item.creators.push({
+					lastName: authors[i].textContent,
+					creatorType: "author",
```

Comment:
"Author" sounds most appropriate, based on how it's used in the byline:
http://tvbythenumbers.zap2it.com/sdsdskdh279882992z1/tv-ratings-friday-grimm-constantine-fall-shark-tank-the-amazing-race-last-man-standing-up-cristela-hawaii-five-0-steady/